core/src/main/java/io/questdb/jit/CompiledFilterIRSerializer.java

```diff
@@ -1029,6 +1034,39 @@ private void serializeIn() throws SqlException {
         }
     }
 
+    private void serializeInTimestampRange(int position) throws SqlException {
+        predicateContext.currentInSerialization = true;
+
+        final CharSequence intervalEx = predicateContext.inOperationNode.rhs.token;
+
+        if (intervalEx == null) return;
+
+        final LongList intervals = new LongList();
```

Comment:
Please introduce a new field for the interval list and reuse it between JIT compilations. There is `LongList#clear()` which resets the list.

Reply:
Done in last commit